Cargo.toml

```diff
@@ -72,21 +72,21 @@ which = { version = "6.0.0", default-features = false }
 zeroize = "1.7.0"
 
 # Dependencies from the `fuel-core` repository:
-fuel-core = { version = "0.23.0", default-features = false }
-fuel-core-chain-config = { version = "0.23.0", default-features = false }
-fuel-core-client = { version = "0.23.0", default-features = false }
-fuel-core-poa = { version = "0.23.0", default-features = false }
-fuel-core-services = { version = "0.23.0", default-features = false }
-fuel-core-types = { version = "0.23.0", default-features = false }
+fuel-core = { version = "0.24.1", default-features = false, features = ["test-helpers"] }
```

Comment:
I'd have these features activated on a need-to-have basis. I.e. have the cargo workspace dep here only to make sure every crate uses (requires) the same fuel-core version but have each crate dictate its own needs as to the features of `fuel-core`. 

Cargo knows how to unify features when compiling a workspace.